src/tiler/server.js

```diff
@@ -154,6 +217,11 @@ var config = {
                 req.params.sql = getSqlForStreamByReq(req);
             }
 
+            if (tableId.indexOf('quality') >= 0) {
```

Comment:
Consider renaming `quality` to `nhd_quality` or `nhd_quality_t` to narrow down the possible matches for this check.